src/main/java/uk/gov/justice/hmpps/prison/service/transformers/OffenderEventsTransformer.java

```diff
@@ -964,6 +965,19 @@ private OffenderEvent iepUpdatedEventOf(final Xtag xtag) {
             .build();
     }
 
+    private OffenderEvent visitCancelledEventOf(final Xtag xtag) {
+        return OffenderEvent.builder()
+            .eventType("VISIT_CANCELLED")
```

Comment:
why convert the update event into a cancelled event?  Will there ever be an update event instead?